packages/block-editor/src/components/convert-to-group-buttons/toolbar.js

```diff
@@ -15,8 +15,12 @@ import { store as blockEditorStore } from '../../store';
 
 const layouts = {
 	group: { type: 'constrained' },
-	row: { type: 'flex', flexWrap: 'nowrap' },
-	stack: { type: 'flex', orientation: 'vertical' },
+	row: { type: 'flex', flexWrap: 'nowrap', allowSizingOnChildren: true },
+	stack: {
+		type: 'flex',
+		orientation: 'vertical',
+		allowSizingOnChildren: true,
+	},
```

Comment:
Hmm, yeah that's a good point. I was thinking in terms of it being a flex-only setting, but we should be able to add it to the block defaults as we always check for the flex layout anyway. I'll give it a try!

Reply:
So this was harder to wire up than i thought it would be because I had to manually loop up the block support and add it to the layout object InnerBlocks passes to BlockListItems (that's the layout that gets used as context for the BlockListBlock). 

The whole thing with block edit functions having to explicitly hand layout down to InnerBlocks is something we wanted to look at as part of stabilising Layout anyway, so I guess we can improve it then?

Reply:
Thanks for following up on that!

> The whole thing with block edit functions having to explicitly hand layout down to InnerBlocks is something we wanted to look at as part of stabilising Layout anyway, so I guess we can improve it then?

The way you've done it right now, passing down in the layout object, sounds like a good short-term approach to me 👍

And yes, good call, I agree we can improve it when we look at stabilising the support, since the change in this PR should be pretty invisible from the perspective of a block's own code.